project/Testing.scala

```diff
@@ -22,6 +23,9 @@ object Testing {
 
   val commonTestSettings: Seq[Setting[_]] = List(
 
+    testOptions in Test += Tests.Setup(() =>
+      sys.props += "mockserver.logLevel" -> "WARN"
```

Comment:
The default log verbosity for mockserver logs every mock defined, for every test that defines it. Since this goes to the console for jenkins jobs, it makes the logs _huge_, to the point that the browser has trouble searching through them.

Reply:
Awesome!!